modules/shortcodes/js/blocks/vr-block.jsx

```diff
@@ -0,0 +1,94 @@
+/* global wp */
+/* eslint react/react-in-jsx-scope: 0 */
+
+( function( blocks, components, i18n ) {
+	const {
+		registerBlockType,
+		UrlInput
+	} = blocks;
+	const {
+		Placeholder,
+		SelectControl
+	} = components;
+	const { __ } = i18n;
+
+	registerBlockType( 'gutenpack/vr', {
+		title: __( 'VR Image' ),
+		icon: 'embed-photo',
+		category: 'embed',
+		supportHTML: false,
+		attributes: {
+			url: {
+				type: 'string',
+			},
+			view: {
+				type: 'string',
+			}
+		},
+
+		edit: props => {
+			const attributes = props.attributes;
+
+			const onSetUrl = value => props.setAttributes( { url: value } );
+			const onSetView = value => props.setAttributes( { view: value } );
+
+			const renderEdit = () => {
+				if ( attributes.url && attributes.view ) {
+					return (
+						<div className={ props.className }>
+							<iframe
```

Comment:
Did you try using `Sandbox` component instead? See: https://github.com/WordPress/gutenberg/tree/master/components/sandbox.
Embed block from core uses it:
https://github.com/WordPress/gutenberg/blob/master/core-blocks/embed/index.js#L247-L251.

It introduces a few fixes which makes it better fit in edit view, e.g. it makes sure that clicking on iframe selects block.

Reply:
That seems nice but doesn't work with the current rendering implementation which shows the live view hosted on `vr.me.sh`, since Sandbox doesn't accept the `src` attribute. I think for now I'll keep the current implementation.

Reply:
Right, it is implemented to pass `html` instead. I wasn't aware 👍